lib/k8s-e2e-tests/tests/vector-agent.rs

```diff
@@ -1214,27 +1187,33 @@ async fn glob_pattern_filtering() -> Result<(), Box<dyn std::error::Error>> {
     let framework = make_framework();
     let override_name = get_override_name(&namespace, "vector-agent");
 
-    let config: &str = &format!(
-        indoc! {r#"
-        kubernetesLogsSource:
-          rawConfig: |
-            exclude_paths_glob_patterns = ["/var/log/pods/{}_test-pod_*/excluded/**"]
-            glob_minimum_cooldown_ms = 5000
-    "#},
-        pod_namespace
-    );
+    const CONFIG: &str = indoc! {r#"
```

Comment:
I can definitely see that some of these tests are just testing that `kubernetes_logs` source works.